templates/taxi/driver_update.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>Update drive licence:</h1>
+  <form action="?next={% url 'taxi:driver-list' %}" method="post">

```

Comment:
The `form` action attribute includes `?next={% url 'taxi:driver-list' %}`. Ensure that this is intentional and correctly redirects after form submission. If not needed, consider removing it.